tests/test_database_manager.py

```
@@ -0,0 +1,368 @@
+import sqlite3
+import unittest
+from unittest.mock import patch
+
+from ambientweather2sqlite.database import (
```

Flake8 (7.2.0)

[error] 'ambientweather2sqlite.database.db_manager' imported but unused

(F401)